reporting/reports.py

```diff
@@ -25,13 +47,19 @@ class Report(discoverable.DiscoverableFeature):
     description = None
     file_name = ""
 
+    # the text to be displayed if not reports are available
+    # the whole template can be override by no_report_templtate
+    no_report_text = "Sorry no reports are currently available"
+
+    # the template to be displayed if there are no reports
+    no_report_template = None
+
     def to_dict(self):
         slug = self.__class__.get_slug()
```

Comment:
Why do we instantiate these ? 
There's no init method, and no per-instance state?